### PR TITLE
Geant4 10.7 physics configurations update

### PR DESCRIPTION
**Pull request by civanch**
#### PR description:
With Geant4 10.7 new features and updated components of Physics List are available. In this PR these updates are included everywhere except production Physics List in order to keep regression.

#### PR validation:
private

#### if this PR is a backport please specify the original PR and why you need to backport that PR:
no

### CODE REVIEW

**Comment by civanch** [comment]
@bsunanda , here 10.7 updates are added except FTFP_BERT_EMM, electromagnetic part affect histories, so left unchanged for the time being.

**Comment by civanch** [comment]
please test

**Comment by civanch** [comment]
please test

**Comment by civanch** [comment]
There is no real problem but simulation histories are changed.

**Comment by christopheralanwest** [comment]
+alca

- Within the AlCa domain, the changes correspond to a trivial renaming

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by civanch** [comment]
please test

**Comment by civanch** [comment]
+1

**Comment by civanch** [comment]
urgent

**Comment by civanch** [comment]
@christopheralanwest, please, resign this PR - I make modification to keep regression, it is clean now.

**Comment by francescobrivio** [comment]
+alca
 - see https://github.com/cms-sw/cmssw/pull/33400#issuecomment-817809599

File: SimG4Core/Application/python/g4SimHits_cfi.py [hunk 1]
```diff
@@ -188,7 +196,7 @@
         EmaxFTFP    = cms.double(25.), # in GeV
         EmaxBERTpi  = cms.double(12.), # in GeV
         LowEnergyGflashEcal = cms.bool(False),
-        LowEnergyGflashEcalEmax = cms.double(100),
+        LowEnergyGflashEcalEmax = cms.double(0.02), # in GeV
```
Comment (silviodonato) [inline SimG4Core/Application/python/g4SimHits_cfi.py:199]: @civanch could you confirm that you changed this value on purpose?

**Comment by silviodonato** [comment]
+1

File: SimG4Core/Application/python/g4SimHits_cfi.py (hunk 1 above)
Comment (civanch) [reply SimG4Core/Application/python/g4SimHits_cfi.py:199]: @silviodonato , yes, I confirm. Thanks for merging of this PR.

**Pull request merged**